README.md

```diff
@@ -80,21 +80,22 @@ spec:
   replicas: 1
   server:
     distribution:
-      name: ollama
+      name: starter
     containerSpec:
-      port: 8321
       env:
       - name: INFERENCE_MODEL
-        value: "llama3.2:1b"
+        value: "llama3.2:3b"
```

Comment:
AFAIK this doesn't need to be changed from `1b` to `3b` - `1b` should continue to work just fine

However, `INFERENCE_MODEL` will need to change to `OLLAMA_INFERENCE_MODEL`, see https://llama-stack.readthedocs.io/en/latest/distributions/self_hosted_distro/starter.html#model-configuration